spec/spec_helper.rb

```diff
@@ -1,15 +1,12 @@
+require "simplecov"
+SimpleCov.start
+
 # This file is copied to spec/ when you run 'rails generate rspec:install'
 ENV["RAILS_ENV"] ||= 'development'
 require File.expand_path("../../config/environment", __FILE__)
 
 require 'coveralls'
 Coveralls.wear!
-require "codeclimate-test-reporter"
-CodeClimate::TestReporter.start
-SimpleCov.formatter = SimpleCov::Formatter::MultiFormatter.new [
-  CodeClimate::TestReporter::Formatter,
-  Coveralls::SimpleCov::Formatter
-]
 
```

Comment:
I believe the above is necessary to get code coverage to show up in CodeClimate as well as coveralls.   It's nice if it's there because you can look at lines called out by static analysis, and then click to see if said lines have code coverage.

https://docs.codeclimate.com/docs/test-coverage-troubleshooting-tips#section--send-coverage-to-both-code-climate-and-coveralls-ruby-only-